index.html

```diff
@@ -261,6 +261,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Seyf O.</p>
+          <p class="contact">
+            <i class="fab fa-facebook"></i>
+            <a href="https://www.facebook.com/profile.php?id=100007625689458" target="_blank">profile.php?id=100007625689458</a>
```

Comment:
Hi, you did everything well, only one thing I recommend you to change.
Your facebook user name is currently shown as "profile.php?id=100007625689458"
I recommend you to change it to your user name, like the suggestion.
You can update it by committing and pushing again, however, if you think that's fine, tell me and I can merge this PR

Thank you for your contribution

```suggestion
            <a href="https://www.facebook.com/profile.php?id=100007625689458" target="_blank">{your facebook username here}</a>
```

Reply:
Thanks for the feedback. I've committed the updated info on the 2nd commitment.